tests/config-spec.js

```diff
@@ -0,0 +1,41 @@
+import '../app/scripts/src/main';
+
+describe('languageSelectConfig', function () {
+
+    beforeEach(function () {
+        angular.mock.module('language-select', (languageSelectConfigProvider) => {
+            this.languageSelectConfigProvider = languageSelectConfigProvider;
+        });
+
+        inject(function (languageSelectConfig) {
+            this.languageSelectConfig = languageSelectConfig;
+        });
+    });
+
+    describe('by default', function () {
+
+        it('should have english language set', function () {
+            expect(this.languageSelectConfig.defaultLanguageId()).toBe('en');
+        });
+
+    });
+
+    describe('configuration', function () {
+
+        it('should allow to change the language list', function () {
+            const languageList = ['language1', 'language2'];
+            this.languageSelectConfigProvider.setAvailableLanguages(languageList);
+
+            expect(this.languageSelectConfig.availableLanguages()).toBe(languageList);
```

Comment:
Would this test be improved by changing `toBe` -> `toEqual`, so the method doesn't need to use the object in memory and the tests can still pass? It doesn't seem that the object being the same in memory is important here.